utils/db_utils.py

```diff
@@ -107,13 +107,14 @@ def query_demographics():
         for col in constants.BINARY_DEMOGRAPHICS_COLS:
             if col in df.columns:
                 df[col] = df[col].apply(str) 
-    columns_to_drop = [col for col in df.columns if col.startswith("metadata")]
-    df.drop(columns= columns_to_drop, inplace=True) 
-    df.drop(columns=['data.xmlResponse', 'data.name', 'data.version', 'data.label'], inplace=True) 
-    modified_columns = perm_utils.get_demographic_columns(df.columns)  
-    df.columns = modified_columns 
-    df.columns=[col.rsplit('.',1)[-1] if col.startswith('data.jsonDocResponse.') else col for col in df.columns]  
-    df.drop(columns=['xmlns:jr', 'xmlns:orx', 'id'], inplace = True) 
+        columns_to_drop = [col for col in df.columns if col.startswith("metadata")]
+        df.drop(columns= columns_to_drop, inplace=True) 
+        modified_columns = perm_utils.get_demographic_columns(df.columns)  
+        df.columns = modified_columns 
+        df.columns=[col.rsplit('.',1)[-1] if col.startswith('data.jsonDocResponse.') else col for col in df.columns]  
+        for col in constants.EXCLUDED_DEMOGRAPHICS_COLS:
+            if col in df.columns:
+                df.drop(columns= [col], inplace=True) 
```

Comment:
why isn't this just 

```suggestion

        df.drop(columns=constants.EXCLUDED_DEMOGRAPHICS_COLS, inplace=True)
```

Reply:
As this columns are not i the data set I am working, we cannot directly exclude all those columns.